libpkpass/commands/generate.py

```diff
@@ -13,19 +13,24 @@ class Generate(Command):
     description = 'Generate a new password entry and encrypt it for yourself'
     selected_args = Command.selected_args + ['pwname', 'pwstore', 'overwrite', 'stdin', 'keypath',
                                              'nopassphrase', 'nosign', 'card_slot', 'escrow_users',
-                                             'min_escrow', 'noescrow', 'rules', 'rules_map']
+                                             'min_escrow', 'noescrow', 'rules', 'rules_map', 'description',
+                                             'authorizer']
 
         ####################################################################
     def _run_command_execution(self):
         """ Run function for class.                                      """
         ####################################################################
         safe, owner = self.safety_check()
+        if not self.args['description']:
+            self.args['description'] = input("Description: ")
+
+        if not self.args['authorizer']:
+            self.args['authorizer'] = input("Authorizer: ")
+
```

Comment:
Is there a reason for placing this outside of the if check? This means a user might get prompted for additional information even if the `NotThePasswordOwnerError` is going to fire.

Reply:
Sorry, I see what you mean.  Added back in the right place